### PR TITLE
update

### PR DESCRIPTION
**Pull request by bobzhang**
- **update**
- **update**
- **fmt**
- **no newline**

### CODE REVIEW

File: json/json.mbti [hunk 1]
```diff
@@ -17,8 +17,7 @@ impl ErrorData {
   to_string(Self) -> String
 }
 
-pub type! ParseError ErrorData
-impl ParseError {
+pub type! ParseError ErrorDataimpl ParseError {
   op_equal(Self, Self) -> Bool
```
Comment (bobzhang) [inline json/json.mbti:21]: newline is missing cc @lynzrand

File: builtin/bigint.mbt [hunk 2]
```diff
@@ -225,11 +225,10 @@ fn grade_school_mul(self : BigInt, other : BigInt) -> BigInt {
...
-      let product = limbs[i + j].to_int64() + self.limbs[i].to_int64() * if j < other_len {
-          other.limbs[j].to_int64()
-        } else {
-          0L
-        } + carry
+      let product = limbs[i + j].to_int64() +
+        self.limbs[i].to_int64() *
+        if j < other_len { other.limbs[j].to_int64() } else { 0L } +
+        carry
       limbs[i + j] = (product % radix).to_int()
```
Comment (bobzhang) [inline builtin/bigint.mbt:232]: vs
```
let product = 
   limbs[i + j].to_int64() + // newline preferred
   self.limbs[i].to_int64() *
   ( if ... else) +  // () preferred
   carrry  
```

**Comment by bobzhang** [comment]
already updated upstream

File: json/json.mbti (hunk 1 above)
Comment (lynzrand) [reply json/json.mbti]: A fix is already under way

**Pull request closed without merging**